.github/workflows/download-link-check.yml

```diff
@@ -0,0 +1,30 @@
+name: Download Link Check
+
+on:
+  deployment_status:
+  schedule:
+    - cron: '0 1-23 * * *'
+
+jobs:
+  run:
+    name: Initialize
+    runs-on: ubuntu-latest
+    if:
+      github.event_name == 'schedule' || (github.event.deployment.ref !=
+      'master' && github.event.deployment_status.state == 'success')
+    steps:
+      - uses: actions/checkout@v2
+      - name: Download Link Checker
+        uses: lycheeverse/lychee-action@v1.0.8
```

Comment:
From what I can tell, `lychee` places the output file inside a `tmp` folder which gets deleted. Without this output file, I'm not sure how we could get the amount of links without creating our own version of `lychee-action`  that doesn't delete the output file.🤔

Reply:
Yes, if errors are found in the links, `lychee-action` creates a `./lychee/out.md`. But since the file is only created if errors are found, we can't use that file to check for amount of links, since we'd have to check for amount of links when there are no errors found.